proxies/live/apiproxy/policies/ServiceCallout.NrlfAuth.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<ServiceCallout async="false" continueOnError="false" enabled="true" name="ServiceCallout.NrlfAuth">
+  <DisplayName>ServiceCallout.NrlfAuth</DisplayName>
+  <Properties/>
+  <Request clearPayload="true" variable="nrlfTokenRequest">
+    <IgnoreUnresolvedVariables>false</IgnoreUnresolvedVariables>
+    <Add>
+      <FormParams>
+        <FormParam name="grant_type">client_credentials</FormParam>
+        <FormParam name="redirect_uri">https://oauth.pstmn.io/v1/callback</FormParam>
+        <FormParam name="client_assertion_type">urn:ietf:params:oauth:client-assertion-type:jwt-bearer</FormParam>
+        <FormParam name="client_assertion">{client_assertion}</FormParam>
+      </FormParams>
+    </Add>
+  </Request>
+  <Response>nrlfTokenResponse</Response>
+  <HTTPTargetConnection>
+    <Properties/>
+    <URL>https://int.api.service.nhs.uk/oauth2/token</URL>
```

Comment:
Same with this, ideally it isn't hardcoded, can we get this (and also redirect_uri?) from the KVM too?